### PR TITLE
LPS-129910 Change CustomTimeRangeForm component to clay

### PR DESCRIPTION
**Pull request by ramonmsp**

### CODE REVIEW

**Comment by ramonmsp** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/filter/CustomTimeRangeForm.es.js [hunk 1]
```diff
@@ -92,43 +93,44 @@ const CustomTimeRangeForm = ({
...
-						/>
-					</FormGroupItem>
-				</div>
+				<ClayForm.Group className="form-group-autofit">
+					<ClayInput.Group>
+						<ClayInput.GroupItem>
+							<FormGroupItem error={errors['dateStart']}>
+								<label htmlFor="dateStart">
+									{Liferay.Language.get('from')}
+								</label>
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/filter/CustomTimeRangeForm.es.js:102]: Hey @ramonmsp , Can you keep the blank line between the siblings tags? Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/filter/CustomTimeRangeForm.es.js (hunk 1 above)
Comment (ramonmsp) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/filter/CustomTimeRangeForm.es.js:102]: Ok!! I'll do it :D

**Comment by ramonmsp** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by ramonmsp** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**